spec/models/authority_spec.rb

```diff
@@ -102,6 +102,48 @@
     end
   end
 
+  describe '#applications_per_week' do
+    let(:authority) { create(:authority) }
+
+    before :each do
+      Timecop.freeze(Time.local(2016, 1, 5))
+    end
+
+    after :each do
+      Timecop.return
+    end
+
+    context "when the authority has no applications" do
+      it "returns an empty list" do
+        expect(authority.applications_per_week).to eq []
+      end
+    end
+
+    context "when the authority has applications" do
```

Comment:
Fantastic ❤️ great to get this covered!

There are a few extra whitespace lines in here compared to our style (123, 125, 144). Could you snip em please? And there’s a line missing between blocks (between 136 and 137) :)